packages/api/lambdas/bulk-operation.js

```diff
@@ -72,14 +72,16 @@ async function applyWorkflowToGranules({
  * @param {string} [payload.index] - Optional parameter of ES index to query.
  * Must exist if payload.query exists.
  * @param {Object} [payload.ids] - Optional list of granule IDs to bulk operate on
- * @param {Function} [unpublishGranuleFunc] - Optional function to delete the
- * granule from CMR. Useful for testing.
+ * @param {Function} [removeGranuleFromCmrFunction] - passed in function used for test mocking
```

Comment:
What is the type of this function?

Reply:
I have slightly mixed feelings generally about how these unit mocks should be typed. 

 That said, it's typed to _removeGranuleFromCmr in the dependent function, which we could import (gross) to correctly type the method, or provide explicit typing.    I'm going with the latter, given TS inclinations, but open to feedback on this.        There's probably a refactor here as well, but that's solidly outside what I'm willing to scope for this ticket, absent strong objections from the team.